apps/backend/src/main.ts

```diff
@@ -65,13 +65,17 @@ async function bootstrap() {
         }),
         proxy: configService.isInProductionMode() ? true : undefined,
         cookie: {
-          maxAge: 60 * 60,
+          maxAge: 60 * 60 * 1000,
```

Comment:
Is this unit milliseconds?

Reply:
As it turns out, yes it is in ms instead of s.

Reply:
Do you want to add a comment or, if the variable name is defined by us, add maxAgeInMsec?

Reply:
I can add a comment.  The parameter is not defined by us.